jest.preset.js

```diff
@@ -0,0 +1,4 @@
+
+      const nxPreset = require('@nrwl/jest/preset').default;
+
+      module.exports = { ...nxPreset }
```

Comment:
Nit: indentation, semi-colons, and EOL are a bit off in this file.

Reply:
Also, should this be a .ts file to match jest.config.ts above?

Reply:
Changed PR to draft.